### PR TITLE
chore: update to @sinet/coding-convetions 2.x

### PR DESCRIPTION
**Pull request by nyan2x**
Closes [TK-09208](https://www12.v1host.com/SchoolImprovementNetwork47/Task.mvc/Summary?oidToken=Task%3A96563&RoomContext=TeamRoom%3A1869)

### CODE REVIEW

**Comment by nyan2x** [comment]
@yamii @openam @sydcanem :) Kindly review my code. Thanks!

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "description": "wrk benchmarking http node.js",
   "main": "index.js",
   "scripts": {
+		"lint": "npm run eslint -- ./ && npm run jscs -- --esnext ./",
```
Comment (yamii) [inline package.json]: fix indention

**Comment by yamii** [comment]
OK :rocket:

**Comment by yamii** [comment]
@openam @sydcanem this repository doesn't have CI running should we add it to `circle-ci`?

**Comment by openam** [comment]
Yeah we should add a `circle.yml` with linting and npm test.

**Pull request merged**